### PR TITLE
feat(Writers Duet): add presence

### LINKED ISSUE #7798: Writer Duet | writerduet.com
### Website name

WriterDuet

### Website URL

https://writerduet.com

### Website logo

https://www.writerduet.com/script/wd-logo-square.png

### Prerequisites

- [ ] It is a paid service
- [ ] It displays NSFW content
- [ ] It is region restricted

### Description

The name of the project being edited and the scene number.

### PR DESCRIPTION
**Pull request by ION606**
## Description 

- Resolves #7798 

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>

image
<summary> Proof showing the creation/modification is working as expected
</summary>

</details>

### CODE REVIEW

**Comment by ION606** [comment]
from https://github.com/PreMiD/Presences/issues/7798

File: websites/W/WriterDuet/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "WriterDuet",
+	"description": {
+		"en": "Plan, write, and share with the industry-standard software used by over 1 million screenwriters, TV shows, and blockbusters"
+	},
+	"url": "writerduet.com",
```
Comment (theusaf) [inline websites/W/WriterDuet/metadata.json:11]:
```suggestion
	"url": "www.writerduet.com",
```

**Review by theusaf — CHANGES_REQUESTED** [review]
It's a bit low-quality. Consider adding other parts of the page (blog, etc) to the presence.

**Comment by ION606** [comment]
> It's a bit low-quality. Consider adding other parts of the page (blog, etc) to the presence.

I just added what the issue asked for

**Comment by Bas950** [comment]
Please add support for other pages

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**